conf.py

```diff
@@ -50,7 +50,7 @@
 
 .. admonition:: Live Notebook
 
-    You can run this notebook in a `live session <https://mybinder.org/v2/gh/dask/dask-examples/main?urlpath=lab/tree/{{ docname }}>`_ |Binder| or view it `on Github <https://github.com/dask/dask-examples/blob/main/{{ docname }}>`_.
+    You can run this notebook in a `live session <https://mybinder.org/v2/gh/dask/dask-examples/main?urlpath=lab/tree/{{ docname }}>`__ |Binder| or view it `on Github <https://github.com/dask/dask-examples/blob/main/{{ docname }}>`__.
```

Comment:
Just for my own understanding, why do we need a double underscore here?

Reply:
The double underscore creates an anonymous link, rather than a link that can be reused. https://docutils.sourceforge.io/docs/user/rst/quickref.html#indirect-hyperlink-targets